content/docs/04-clusters/03-edge/11-deploy-cluster.md

```diff
@@ -0,0 +1,856 @@
+
+
+|**Pack Type**|**Registry**|**Pack Name**|**Pack Version**| 
+|---|---|---|---|
+|Network|Public Repo|Calico|`3.25.x`|
+
+
+Finally, click on the **Confirm** button to complete the core infrastructure stack. Palette displays the newly created infrastructure profile as a layered diagram. 
+
+Next, click on the **Add Manifest** button on the top to add the *Hello Universe* application manifest.  
+
+![A screenshot of the add Manifest button.](/tutorials/edge-native/clusters_edge_deploy-cluster_add-manifest.png)
+
+Use the following values to add the Hello Universe manifest metadata.  
+
+|**Field** |**Value**|
+|---|---|
+|Layer name| hello-universe|
+|Layer values (Optional)|Leave default|
+|Install order (Optional)|Leave default|
+|Manifests|Add new manifest, and name it `hello-universe`|
+
+Entering `hello-universe` file name in the **Manifest** field will open a blank file in Palette's text editor. Copy the following manifest and paste it in Palette's text editor. 
+<br />
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: hello-universe-service
+spec:
+  type: NodePort
+  selector:
+    app: hello-universe
+  ports:
+  - protocol: TCP
+    port: 8080
+    targetPort: 8080
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hello-universe-deployment
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: hello-universe
+  template:
+    metadata:
+      labels:
+        app: hello-universe
+    spec:
+      containers:
+      - name: hello-universe  
+        image: ghcr.io/spectrocloud/hello-universe:1.0.12     # Static image 
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 8080
+```
+
+The screenshot below shows the manifest pasted into the text editor. Click on the **Confirm & Create** button to finish adding the manifest. 
+
+  ![A screenshot of Hello Universe application manifest.](/tutorials/edge-native/clusters_edge_deploy-cluster_add-manifest-file.png)
+
+The manifest above defines two Kubernetes objects:
+<br />
+
+1. **Deployment**: 
+It pulls a public image, **ghcr.io/spectrocloud/hello-universe:1.0.12**, and creates a ReplicaSet to bring up two pods for the `hello-universe` application.
+
+
+2. **Service**: It uses NodePort to expose the `hello-universe` application on each Node at a random port between 30000-32767. 
+<br />
+
+
+If there are no errors or compatibility issues, Palette displays the newly created full cluster profile. Verify the layers you added, and click on the **Next** button. 
+<br />
+
+### Review
+Review all layers and click **Finish Configuration** to create the cluster profile. 
+<br />
+
+## Create a Cluster
+Click on the newly created cluster profile to view its details page. Palette displays all the layers again and allows you to edit any of them. Click on the **Deploy** button and refer to the [Create an Edge Native Host Cluster](/clusters/edge/site-deployment/site-installation/cluster-deployment) guide to deploy a new Edge cluster. 
```

Comment:
```suggestion
Click on the newly created cluster profile to view its details page.  Click the **Deploy** button to deploy a new Edge cluster. 
```

The refer statement will cause someone to leave the guide.  Best to remove it. If you want to mention it, use an infobox. But ask yourself first, is it needed? And does it make sense to introduce that here?